Dockerfile.ci

```diff
@@ -0,0 +1,22 @@
+ARG image_tag=develop
+FROM elifesciences/proofreader-php_composer_dev:${image_tag} AS build
+FROM elifesciences/php_7.0_cli:656bb4bdf1e49a5e80337e2a7c4f44f10c3f52b0
+
+ENV PROJECT_FOLDER=/srv/proofreader-php
+
+USER root
+
+WORKDIR ${PROJECT_FOLDER}
+RUN touch .php_cs.cache && \
+    chown elife:elife . && \
+    chown www-data:www-data .php_cs.cache
+
+USER elife
+
+COPY --chown=elife:elife .php_cs ./
```

Comment:
the ownership of files in the `ci` image looks completely different from the published image, so should these files be owned by root to promote test and production parity?